src/toast/scripts/toast_merge_timings.py

```diff
@@ -37,41 +38,40 @@ def process_timer_path(s):
     :return: A tuple containing the simplified path, kernel type, and operation name.
     """
     # Remove occurrences of '(function) ' and '._exec'
-    simplified_path = s.replace("(function) ", "").replace("._exec", "")
+    simplified_path = s.replace('(function) ', '').replace('._exec', '')
```

Comment:
Something is changing the quotes here.  The black formatter should keep everything as double quotes, which is the desired behavior unless there are nested quotes.

Reply:
Okay, I will run the formatter!